operator/build/build_kvmoperator.sh

```diff
@@ -32,27 +32,27 @@ fi
 # remove old images
 docker images | grep kvmsoperator | awk '{print $3}' | xargs -I {} docker rmi -f {} 2> /dev/null
 
-echo "[INFO] Removed existing kvmsoperator/kvmsoperator images"
+echo "[INFO] Removed existing accuknox/kvmsoperator images"
 
 # remove old files (just in case)
 $ARMOR_HOME/build/clean_source_files.sh
 
-echo "[INFO] Removed source files just in case"
+echo "[INFO] Removed existing source files"
 
 # copy files to build
 $ARMOR_HOME/build/copy_source_files.sh
 
 echo "[INFO] Copied new source files"
 
 # build a new image
-echo "[INFO] Building kvmsoperator/kvmsoperator:$VERSION"
+echo "[INFO] Building accuknox/kvmsoperator:$VERSION"
```

Comment:
```suggestion
echo "[INFO] Building kubearmor/kvmsoperator:$VERSION"
```

Reply:
Which dockerhub will be used? Kubearmor or accuknox dockerhub ?

Reply:
we have kubearmor dockerhub that we can use.